test/unit/services/auto-approval/test-auto-approval-process.js

```diff
@@ -13,20 +13,22 @@ var validAutoApprovalData = testHelper.getAutoApprovalData(reference1)
 var validCheckResult = new AutoApprovalCheckResult('', true, '')
 
 var getDataForAutoApprovalCheckStub = sinon.stub().resolves(validAutoApprovalData)
-var areChildrenUnder18Stub = sinon.stub().resolves(validCheckResult)
-var costAndVarianceEqualOrLessThanFirstTimeClaimStub = sinon.stub().resolves(validCheckResult)
-var doExpensesMatchFirstTimeClaimStub = sinon.stub().resolves(validCheckResult)
-var hasClaimedLessThanMaxTimesThisYearStub = sinon.stub().resolves(validCheckResult)
-var hasUploadedPrisonVisitConfirmationAndReceiptsStub = sinon.stub().resolves(validCheckResult)
-var isClaimSubmittedWithinTimeLimitStub = sinon.stub().resolves(validCheckResult)
-var isClaimTotalUnderLimitStub = sinon.stub().resolves(validCheckResult)
-var isLatestManualClaimApprovedStub = sinon.stub().resolves(validCheckResult)
-var isNoPreviousPendingClaimStub = sinon.stub().resolves(validCheckResult)
-var isVisitInPastStub = sinon.stub().resolves(validCheckResult)
-var visitDateDifferentToPreviousClaimsStub = sinon.stub().resolves(validCheckResult)
+var autoApproveClaim = sinon.stub().resolves()
+var areChildrenUnder18Stub = sinon.stub().returns(validCheckResult)
+var costAndVarianceEqualOrLessThanFirstTimeClaimStub = sinon.stub().returns(validCheckResult)
+var doExpensesMatchFirstTimeClaimStub = sinon.stub().returns(validCheckResult)
+var hasClaimedLessThanMaxTimesThisYearStub = sinon.stub().returns(validCheckResult)
+var hasUploadedPrisonVisitConfirmationAndReceiptsStub = sinon.stub().returns(validCheckResult)
+var isClaimSubmittedWithinTimeLimitStub = sinon.stub().returns(validCheckResult)
+var isClaimTotalUnderLimitStub = sinon.stub().returns(validCheckResult)
+var isLatestManualClaimApprovedStub = sinon.stub().returns(validCheckResult)
+var isNoPreviousPendingClaimStub = sinon.stub().returns(validCheckResult)
+var isVisitInPastStub = sinon.stub().returns(validCheckResult)
+var visitDateDifferentToPreviousClaimsStub = sinon.stub().returns(validCheckResult)
 
 var autoApprovalChecks = {
```

Comment:
add a test case for when one of the approval checks is false to check it does not mark as approved